packages/fxa-settings/src/models/Account.ts

```diff
@@ -641,6 +641,10 @@ export class Account implements AccountData {
     firefox.profileChanged({ uid: this.uid });
   }
 
+  setLastLogin(date: number) {
```

Comment:
Content server was calling `setSignedInAccount` after a user reset their password. If you look at the implementation of this, you'll see the lastLogin field on the account was set. Maybe we don't need to do this, but it seems like it could be an important state.

_I didn't see away to do this in settings currently._